python-sdk/src/langwatch/telemetry/__tests__/test_tracing.py

```
@@ -0,0 +1,48 @@
+import pytest
+from langwatch.telemetry.tracing import LangWatchTrace
+
+
+def test_metadata_initialization_and_update_merge():
+    # Initial metadata
+    trace = LangWatchTrace(metadata={"foo": 1, "bar": 2})
+    assert trace.metadata == {"foo": 1, "bar": 2}
```

Ruff (0.12.2)

Use of `assert` detected

(S101)